### DISCUSSION TITLE
关于NETEASE_COOKIE的调用问题

### DISCUSSION THREAD

**Discussion by M-HLSLY** [1715173329/UnblockNeteaseMusic#1391] [Q&A] [answered]
在“环境变量”看到了相关信息，但不知道做什么用，可否被音源调用？

**Comment by pan93412** [comment, marked as answer]
看起來是用來切換到其他帳號播放 VIP 歌曲，不過根據 PR 來看用途沒有很大 https://github.com/UnblockNeteaseMusic/server/pull/1342
